packages/@react-types/table/src/index.d.ts

```diff
@@ -10,72 +10,122 @@
  * governing permissions and limitations under the License.
  */
 
-import {AriaLabelingProps, AsyncLoadable, CollectionChildren, DOMProps, LoadingState, MultipleSelection, SectionProps, Sortable, StyleProps} from '@react-types/shared';
+import {AriaLabelingProps, AsyncLoadable, CollectionChildren, DOMProps, LoadingState, MultipleSelection, Sortable, StyleProps} from '@react-types/shared';
 import {GridCollection, GridNode} from '@react-types/grid';
 import {Key, ReactElement, ReactNode} from 'react';
 
 export interface TableProps<T> extends MultipleSelection, Sortable {
-  children: ReactElement<TableHeaderProps<T> | TableBodyProps<T> | SectionProps<T> | RowProps<T>>[],
+  // TODO: I think this shouldn't have Section and row? Also need to fix prop table rendering for this
```

Comment:
i think this TODO makes sense, though should it really be?
`children: [ReactElement<TableHeaderProps<T>>, ReactElement<TableBodyProps<T>>]`
I don't know if a table can have only one of those, but seems like there shouldn't be more than one of each?